packages/repository/src/__tests__/fixtures/mixins/categoryPropertyMixin.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Constructor} from '@loopback/context';
+import {property} from '../../../index';
```

Comment:
Please either remove `index` (import from `'../../..`), or change the import to reference the specific file implementing the decorator (I think that would be `'../../../src/decorators/model.decorator`).